src/browser/components/molecules/Task/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import Layout from '../../molecules/Layout/index'
+import styles from './index.css'
+const moment = require('moment')
+
+
+
+export default class Task extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      complete: false
+    }
+    this.isCompleteHandler = this.isCompleteHandler.bind(this)
+  }
+
+  isCompleteHandler( event ){
+    const completeTaskDetails = {
+      method: 'POST',
+      mode: 'cors',
+      headers: new Headers({
+        'Accept': 'application/json, application/xml, text/plain, text/html, *.*',
+        'Content-Type': 'application/json',
+         }),
+      credentials: 'same-origin',
+      body: JSON.stringify({is_complete: true})
+    }
+    fetch( `http://noob.learnersguild.dev/api/task/${this.props.id}`, completeTaskDetails )
+    .then( resultPromise => {
+      return resultPromise.json()
+    })
+    .then( result => {
+      this.props.loadTasks()
```

Comment:
this function `this.props.loadTasks` should take the argument `result`. Looking at the definition of `loadTasks` it is also making the call to fetch the tasks, which is wasted effort. The responsibility of the function `loadTasks` should only be to update the state. Hence the name `loadTasks` wouldn't fit the behavior. Change the name to something like `updateTasksCallback`